### DISCUSSION TITLE
WarsawJS meetup 08.11.2023

### DISCUSSION THREAD

**Discussion by pwierzbieniec** [Git-Prove/GitProve-Projects#7] [Events & Meetups]
**Date**

08.11.2023 

**Time**

18:30

**Place**

Paramount, Mokotowska 19, 00-560 Warsaw

**Agend**

► Daniel Janus — Cross-pollinating ideas #1: Speccing data

► Dzmitry Krystsek — Processing big data files with Node.js

► Jerzy Nowiński — How (not)to optimize web applications

**Tickets**

 https://app.evenea.pl/event/warsawjs-meetup-109

**Comment by LachPawel** [comment]
Looks interesting, especially second topic as it's always a tricky thing to do in Node.js - anyone going?
